src/main/java/io/commercetools/sunrise/pages/PDP.java

```diff
@@ -0,0 +1,47 @@
+package io.commercetools.sunrise.pages;
+import org.fluentlenium.core.Fluent;
+import org.fluentlenium.core.FluentPage;
+import com.typesafe.config.ConfigFactory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class PDP extends FluentPage {
```

Comment:
Yes, I'm trying to remove all "PDP" and "POP" references from open-source projects, since not everybody understands the term.